projects/elife.yaml

```diff
@@ -1179,6 +1159,9 @@ elife-alfred-nodes:
     domain: false
     intdomain: false
     aws:
+        # 16.04, deprecated
+        ami: ami-14a1d06e # GENERATED created from basebox--1604
```

Comment:
of course, projects that are already on 16.04 should stay pinned to that rather than 14.04, waiting for their 18.04 upgrade